react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(id) {
+    if (this.state[id] === undefined) {
+      this.setState({ [id]: true });
+    } else {
+      this.setState({ [id]: !this.state[id] });
+    }
+
+  }
```

Comment:
This will create a bunch of properties in state, it is better to just have a single property in state for this exercise.
```JAVASCRIPT
handleClick(id) {
  if (this.state.openId === id) {
    this.setState({ openId: null });
  } else {
    this.setState({ openId: id });
  }
}
```